com.unity.formats.alembic/CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 ### Changed
 ### Fixed
+- Fixed a bug that caused unintended end of range stream edits.
```

Comment:
Suggestion:
`Fixed an issue where changing the start value of the streaming time range would slightly change the end of range value.`